src/renderer/pages/player-page.js

```diff
@@ -497,7 +497,7 @@ function renderPlayerControls (state) {
       <div
         key='scrub-bar'
         className='scrub-bar'
-        draggable='true'
+        draggable
```

Comment:
I not sure but I think that we should leave it as it was according to [draggable documentation](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/draggable)

Reply:
The jsx draggable type is boolean (not string), and this is shorthand for `draggable={true}`. I believe that should be correct according to the docs.